qiskit/visualization/gate_map.py

```diff
@@ -425,28 +428,31 @@ def plot_error_map(backend, figsize=(12, 9), show_title=True):
     cmap = config['coupling_map']
 
     directed = False
-    if n_qubits < 20:
-        for edge in cmap:
-            if not [edge[1], edge[0]] in cmap:
-                directed = True
-                break
-
-    cx_errors = []
-    for line in cmap:
-        for item in props['gates']:
-            if item['qubits'] == line:
-                cx_errors.append(item['parameters'][0]['value'])
-                break
-        else:
-            continue
+    line_colors = []
+    if cmap:
+        directed = False
```

Comment:
Is this line needed with L430 above?

Reply:
Ok yeah. Soon all devices will be non directed so can remove shortly.